### DISCUSSION TITLE
How can i backup my Floorp Notes, & also then copy them to other Floorp installations?

### DISCUSSION THREAD

**Discussion by guigirl42** [Floorp-Projects/Floorp#1078] [Q&A] [answered]
I deliberately do NOT use Sync, in any browser. Hence please don't reply that i can solve this objective by using it. I'd like to be able to do it manually. 

I know of file `floorp_notes_backup.json` in my profile directory, & initially assumed that copying this to other Floorp installations would thus give me my notes in other computers. However it does not work. This file is not getting any updates; it contains none of my recent Notes at all, only very old ones from last year. 

Where are all my Notes _actually_ being stored please? Which file or directory do i need to copy?

**Comment by creeper-0910** [comment, marked as answer]
about:config→floorp.browser.note.memos

**Reply by guigirl42** [reply]
Hi, t/y for the fast reply! Sorry but i don't understand. I've now searched `about:config` for the string `flooorp.browser.note.memos` but that string does not exist...

...oh, haha, now i see, you made a typo, one `o` too many :smile: , if i instead search for `floorp.browser.note.memos` then yes now i see all the text. 

Is there no actual specific individual file holding all this content?

**Reply by guigirl42** [reply]
I realise now that my question
>Is there no actual specific individual file holding all this content?

was naive. Ofc all that data, like all other `about:config` data, will be within my `prefs.js` file... along with LOTS of other _unrelated_ settings & data. 

Now that you've given me this info, until a better alternative appears, i have copied all that text within `floorp.browser.note.memos` to an external text file, which is thus "backup-able". Next time i need to share this profile's Notes with another profile, i'll just copy all this text then paste it into that profile's `about:config` `floorp.browser.note.memos`. It's cumbersome & inelegant, but i expect it will work fine. 

I've now Closed this Question. Thanks.

**Reply by creeper-0910** [reply]
Oh lol, that is a typo.
Glad you were able to resolve it!
